pkgs/development/python-modules/releases/default.nix

```
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, semantic-version
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "releases";
+  version = "1.6.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "bitprophet";
+    repo = pname;
+    rev = version;
+    hash = "sha256-XX2e6bjBNMun31h0sNJ9ieZE01U+PhA5JYYNOuMgD20=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "semantic_version<2.7" "semantic_version"
+  '';
+
+  propagatedBuildInputs = [ semantic-version sphinx ];
+
+  # No tests...
```

The repos contains a folder [tests](https://github.com/bitprophet/releases/tree/master/tests). IT doesn't seem to be a test suite in the classical sense.

```suggestion
  # No tests
  doCheck = false;

  pythonImportsCheck = [
    "releases"
  ];
```

---

Addressed in https://github.com/NixOS/nixpkgs/commit/82761fb688373e2ca6f08b1f4e718a4f5859eaba and https://github.com/NixOS/nixpkgs/commit/5e7bced95e9e0114653435528c1cdb8e03d48491.